### PR TITLE
Files and directories from private repository

### PR DESCRIPTION
**Pull request by sd-lueckel**

### CODE REVIEW

File: tests/test.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+- hosts: localhost
+  remote_user: root
+  roles:
+    - solutiondrive.deployment
```
Comment (michz) [inline tests/test.yml]: Newline

File: tests/inventory [hunk 2]
```diff
@@ -0,0 +1 @@
+localhost
```
Comment (michz) [inline tests/inventory]: Newline

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+  # Create ansible.cfg with correct roles_path
+  - printf '[defaults]\nroles_path=../' >ansible.cfg
+
+script:
+  # Basic role syntax check
+  - ansible-playbook tests/test.yml -i tests/inventory --syntax-check
+
+notifications:
+  webhooks: https://galaxy.ansible.com/api/v1/notifications/
```
Comment (michz) [inline .travis.yml]: Newline

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1 @@
+.idea/
```
Comment (michz) [inline .gitignore]: NEwline

**Review by michz — CHANGES_REQUESTED** [review]

**Review by michz — APPROVED** [review]

**Review by Jnoack331 — APPROVED** [review]

**Review by mattagohni — APPROVED** [review]

**Comment by michz** [comment]
Here some permissions are not correct yet, @solutionDrive/maintainer is not allowed to merge a PR.

**Pull request merged**